AgenciaAutos/AgenciaAutos/Classes/Client.cs

```diff
@@ -0,0 +1,32 @@
+﻿using AgenciaAutos.Classes;
+using System;
+
+namespace AgenciaAutos
+{
+    class Client : IEntity
+    {
+        public int Id { get; set; }
+        public string name { get; set; }
+        public string lastName { get; set; }
+        public string telephoneNumber { get; set; }
+        public string address { get; set; }
+        public string city { get; set; }
+        public string state { get; set; }
+        public int postalCode { get; set; }
+        public DateTime lastModified { get; set; }
+
+        public Client() { }
+        public Client(int _id,string _name, string _lastname, string _telephoneNumber, string _address, string _city, string _state, int _postalCode, DateTime _lastModified)
```

Comment:
same comment applies here, watch out for casing issues and always try to follow conventions!

Reply:
yes, i´m trying to adapt me to write with c# conventions.